pom.xml

```diff
@@ -173,6 +179,12 @@
             <version>1.9.0</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.sling</groupId>
+            <artifactId>org.apache.sling.serviceusermapper</artifactId>
+            <version>1.5.8</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
do you still need that dependency?

Reply:
no i don't think so. 👍 will check and remove it

Reply:
removed. thanks for spotting. that's awesome.